File: Fibonachi/fibonachi.pl
sub Fibonachi(){
	
	$first = 0;
	$second = 1;
	$fib = 0;

	for ($item = 0; $item <= 10; $item++){
		print"[!]- This is [FIRST] number --> ",$first,"\n";
		print"[!]- This is [SECOND] number --> ",$second,"\n";
		$fib = $first+$second;
		print"\t[+]---> This is [FIB] number --> ",$fib,"\n";
		$first = $second;
		$second = $fib; 
	}
}

Fibonachi();

File: Fibonachi/fibonachi.go
package main

import "fmt"

func main(){
	first, second, fib  := 0,1,0
	for item:=0; item <= 10; item+=1{
		fmt.Printf("[-]- This is [FIRST] number --> %d\n", first)
		fmt.Printf("[-]- This is [SECOND] number --> %d\n", second)
		fib = first+second
		fmt.Printf("[+]---> This is [FIBONACHI] number --> %d\n", fib)
		first, second = second, fib
	}
}
